README.md

```diff
@@ -117,7 +117,8 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+rails new tunr: creates a new rails app called 'tunr'
+rails new . -d postresql: creates a new rails app using postgresql
```

Comment:
the difference is the `.` means create the app in the currently directory using the current directory as the name for the app.